README.md

```diff
@@ -57,7 +58,7 @@ This package requires the Airbnb font and the extrafont package.
 
 ```r
 # configured to work on a Mac, change directory to Unix or Windows
-download.file("https://github.com/ricardo-bion/ggtech/blob/master/Circular%20Air-Light%203.46.45%20PM.ttf", "~/Circular Air-Light 3.46.45 PM.ttf", method="curl")
```

Comment:
@ abidawson The fonts are now on Github. Can you fix this last one thing, and then we are ready to merge? Thanks!

Reply:
@ricardo-bion Should be fixed now, if not (or anything else is wrong) let me know. Thanks!